### DISCUSSION TITLE
Line botからスプレッドシートへの書き込みがされていない

### DISCUSSION THREAD

**Discussion by De-chi** [okuokuch/pokemon_sv_vs_support#118] [Q&A]
お世話になっております。以前LINEBotの自動応答がされないことで相談した者です。
あの後、LINEBotは自動応答されるようになったのですが、ポケモンのパーティをBotに入力してもスプレッドシートに書き込み完了とは出るのですが、実際に書き込みされていないようです。
こちら、どのような原因が考えられますでしょうか。
ご確認お願いします。

**Comment by okuokuch** [comment]
@De-chi 
現状だと原因が絞り込めないので、詳細を教えていただきたいです。
以下2点について、実際の文章をお教えください！
- 送ったメッセージ
- LINEbotから返ってきたメッセージ

**Reply by De-chi** [reply]
@okuokuch 
送ったメッセージ→ポケモン6体（おくおくさんのLineBotで反映されたポケモン6体をコピペ）
帰ってきたメッセージ→ スプレッドシートに書き込み完了
対戦id1001に追加しました。

**Reply by okuokuch** [reply]
@De-chi 
ありがとうございます。
こちらバグで、別途対応中です。バグが治るまでは以下対応をしていただければと思います。
【対処】
- 1001行目にデータが格納されているか確認。データが格納されている場合は(複数回実行した場合は1002行目以降もすべて)削除してください。
- 2行目にダミーとして適当なデータを入力してください。
  - 1001行目で入力されていた値をコピペでも大丈夫です。
- これ以降は3行目から追加されることを確認。

バグの詳細が知りたい場合はこちらをご覧ください。#117
